cluster/examples/tectonic/matchbox.cloud-config

```diff
@@ -0,0 +1,106 @@
+#cloud-config
```

Comment:
Cloud configs are deprecated and should be replaced with Ignition files, see https://coreos.com/os/docs/latest/cloud-config.html.

Reply:
The problem is (just like GCE) that XenServer still relies on cloud-config files. Ignition is not yet implemented. That's why I still use it.